.github/scripts/update-latests-per-major-version.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+version=$1
+
+if [[ $version =~ v([0-9]+)\.([0-9]+)\.([0-9]+)-([0-9]+)-nightly ]]; then
+    msb=${BASH_REMATCH[1]}
```

Comment:
> msb=${BASH_REMATCH[1]}

This matches the first number (major). I guess you can just use `$version` - no BASH_REMATCH needed.

```bash
filename="latest_${version}"
```